### ISSUE TITLE
Don't get different file type formats when uploading new images

### ISSUE THREAD

**Issue by JojokCreator** [nicolashmln/strapi-plugin-responsive-image#48]
![image](https://github.com/nicolashmln/strapi-plugin-responsive-image/assets/98471049/6dcd3d2e-6bf4-44b4-ac93-de7f8714bf31)

I've installed the plugin and rebuilt the server. When I add a new image to the media gallery I don't get the different file formats only the large,small, medium and thumbnail of the uploaded file format.

Am I missing something?

**Comment by svenruegg** [comment]
Have you seen, that you need to add a small code snippet as well?
Check the readme.

**Comment by mauromontano** [comment]
Same problem here, I've installed the plugin and added the strapi-server config but is not working after uploading a new image, any idea? I have the 4.24.2v

**Comment by svenruegg** [comment]
@mauromontano does the plugin show up in the settings of strapi admin panel?

**Comment by mauromontano** [comment]
@svenruegg yes, I have the different formats and this config in the media library, should the image be formatted after uploading from the media library?

![Screenshot 2024-08-14 at 4 46 50 PM](https://github.com/user-attachments/assets/72b0faf9-84db-4245-a5f3-c6881b32d4f2)

![Screenshot 2024-08-14 at 4 47 00 PM](https://github.com/user-attachments/assets/4f9f6c79-5bce-4be2-9c85-27ff61973e81)

**Comment by lan23333** [comment]
same issue here, any updates on this issue?
